rules-reviewed/core.windup.categories.xml

```diff
@@ -15,4 +15,24 @@
         <description>Items in this category will need to be assessed manually to determine whether or not action is required.</description>
     </category>
 
+    <category id="cloud-mandatory" priority="1010">
+        <name>Cloud Mandatory</name>
+        <description>Items that will require changes in order for the migration to be successful.</description>
+    </category>
+
+    <category id="cloud-optional" priority="2010">
+        <name>Cloud Optional</name>
```

Comment:
yeah: more descriptive is better ;-)  
I change the 4 new categories' names to "(Mandatory|Optional) for (OpenShift|a cloud environment)"